src/cnxml/jsx.ts

```diff
@@ -42,15 +42,15 @@ export interface ProcessingInstruction {
 export type Node = Element | ProcessingInstruction | globalThis.Node | string | Node[] | null
 
 export const Node = {
-    nodeIsNotJSXElement(node: Node):
+    isNotJSXElement(node: Node):
         node is ProcessingInstruction | globalThis.Node | string | Node[] | null {
         return typeof node === 'string'
             || node === null
             || Array.isArray(node)
             || node instanceof globalThis.Node
-            || Node.nodeIsProcessingInstruction(node)
+            || Node.isProcessingInstruction(node)
```

Comment:
I would prefer a positive test (`isElement`). It will be simpler to read and won't need to be updated if we ever add another type of `Node`.